tests/boardfarm_plugins/boardfarm_prplmesh/tests/prplmesh_base_test.py

```diff
@@ -30,3 +32,143 @@ def prplmesh_status_check(self, entity_or_radio: Union[env.ALEntity, env.Radio])
+            "No CMDU <msg> found".
+
+        msg_type: int
+            CMDU message type that is expected.
+
+        eth_src: str
+            MAC address of the sender that is expected.
+
+        eth_dst: str
+            MAC address of the destination that is expected. If omitted, the IEEE1905.1 multicast
+            MAC address is used.
+
+        mid: int
+            Message Identifier that is expected. If omitted, the MID is not checked.
+
+        Returns
+        -------
+        [sniffer.Packet]
+            The matching packets.
+        """
+        debug("Checking for CMDU {} (0x{:04x}) from {}".format(msg, msg_type, eth_src))
+        result = self.dev.DUT.wired_sniffer.get_cmdu_capture_type(msg_type, eth_src, eth_dst, mid)
+        assert result, "No CMDU {} found".format(msg)
+        return result
+
+    def check_cmdu_type_single(
+        self, msg: str, msg_type: int, eth_src: str, eth_dst: str = None, mid: int = None
+    ) -> sniffer.Packet:
+        '''Like check_cmdu_type, but also check that only a single CMDU is found.'''
+        cmdus = self.check_cmdu_type(msg, msg_type, eth_src, eth_dst, mid)
+        assert len(cmdus) == 1, "Multiple CMDUs {} found".format(msg)
+        return cmdus[0]
+
+    def check_no_cmdu_type(
+        self, msg: str, msg_type: int, eth_src: str, eth_dst: str = None
+    ) -> NoReturn:
+        '''Like check_cmdu_type, but check that *no* machting CMDU is found.'''
+        debug("Checking for no CMDU {} (0x{:04x}) from {}".format(msg, msg_type, eth_src))
+        result = self.dev.DUT.wired_sniffer.get_cmdu_capture_type(msg_type, eth_src, eth_dst)
+        if result:
+            for packet in result:
+                debug("  {}".format(packet))
+            assert False, "Unexpected CMDU {}".format(msg)
+
+    def check_cmdu_has_tlvs(
+        self, packet: sniffer.Packet, tlv_type: int
+    ) -> [sniffer.Tlv]:
+        '''Check that the packet has at least one TLV of the given type.
+
+        Mark failure if no TLV of that type is found.
+
+        Parameters
+        ----------
+        packet: Union[sniffer.Packet, None]
+            The packet to verify. It may also be None, to make it easy to let it follow
+            check_cmdu_type_single. If it is None, no failure is raised. If it is not an IEEE1905
+            packet, a failure *is* raised.
+
+        tlv_type: int
+            The type of TLV to look for.
+
+        Returns
+        -------
+        [sniffer.Tlv]
+            List of TLVs of the requested type. Empty list if the check fails.
+        '''
+        assert packet, "No packet found"
+        assert packet.ieee1905, "Packet is not IEEE1905: {}".format(packet)
+        tlvs = [tlv for tlv in packet.ieee1905_tlvs if tlv.tlv_type == tlv_type]
+        if not tlvs:
+            debug("  {}".format(packet))
+            assert False, "No TLV of type 0x{:02x} found in packet".format(tlv_type)
```

Comment:
```suggestion
        assert tlvs, "No TLV of type 0x{:02x} found in packet: {}".format(tlv_type, packet)
```

does the debug print go to separate output? wouldn't the above be enough here?

Reply:
Yes, we don't want to see debug output in CI logs for example (I guess). It prints more verbose information and large chunks of packet data or stack traces etc.